CHANGELOG.md

```diff
@@ -27,12 +24,11 @@ The Lotus V2 APIs introduce a powerful new TipSet selection mechanism that signi
 > [!NOTE]
 > V2 APIs are highly experimental and subject to change without notice.
 
-> [!IMPORTANT]
-> The `/v2` APIs rely on F3 data even if F3 is not yet finalizing the chain (i.e., `EC.Finalize` is `false` in the live F3 manifest used by all participants). To determine if F3 is actively finalizing, call the `F3GetManifest` API and check if `Manifest.EC.Finalize` is `true`. Only when `EC.Finalize` is `true` will the `/v2` `"finalized"` and `"safe"` tags accurately reflect the chain's finality according to F3. This also applies to Ethereum APIs discussed below.
```

Comment:
This was moved to https://github.com/filecoin-project/lotus/pull/13088/commits/977d49e559ddf618506504f3337959297d98d5b9